src/NzbDrone.Core/Update/Commands/InstallUpdateCommand.cs

```diff
@@ -0,0 +1,13 @@
+using NzbDrone.Core.Messaging.Commands;
+
+namespace NzbDrone.Core.Update.Commands
+{
+    public class InstallUpdateCommand : Command
+    {
+        public override bool SendUpdatesToClient => true;
+        public override bool IsExclusive => true;
+
+        public override string CompletionMessage => null;
+        public UpdatePackage LatestUpdate;
```

Comment:
Should we make this a property instead?

Reply:
Yea probably, also feel free to push to this PR. Just wanted to get something up for discussion before bed, wasn’t sure what time I’d have today.

Reply:
Fixed up.